src/constants.js

```diff
@@ -100,6 +100,9 @@ export const ROOT_TOKEN = '__ROOT__'
 // token for hidden system context
 export const EM_TOKEN = '__EM__'
 
+// token for recentlyEdited tree nodeChange special case (add)
+export const DUMMY_TOKEN = ''
```

Comment:
It would be better to use a value of `__DUMMY__` as it is less likely to conflict with real values.

This can also be defined in `recentlyEditedTree.js` since it is only used in that one file.

Reply:
Sure

Reply:
I meant `const DUMMY_TOKEN = '__DUMMY__'`. Sorry for the confusion.

Reply:
Done!